strings.txt

```diff
@@ -5748,6 +5748,10 @@ OUTPUT_CHANNELS_RIGHT
 	PL	Kanał prawy
 	RU	Правый канал
 	SV	Höger kanal
+	
+OUTPUT_CHANNELS_COMBINED
+	EN	Combined 
+	FR	Combinés
```

Comment:
Ok, I trust you for the technical implementation. But the wording?... 😁 

Shouldn't this be called something like "Both Channels (Mono)" (like for the default mode)?

Reply:
You should not trust either :-) If you think that Both Channels is clearer, then that's fine with me of course

Reply:
It would be more in line with the other settings. "Mono" probably is the important part, as that's what I as a layman do understand.